Tests/Library/Client.tests.ts

```diff
@@ -186,55 +192,55 @@ describe("Library/Client", () => {
                 host: "bing.com"
             }
         };
-        
+
         var response = {
-            emitFinish: function(): void {
+            emitFinish: function (): void {
                 if (this.finishCallback) {
                     this.finishCallback();
                 }
             },
-            once: function(event: string, callback: Function): eventEmitter.EventEmitter {
+            once: function (event: string, callback: Function): eventEmitter.EventEmitter {
                 if (event === 'finish') {
                     this.finishCallback = callback;
                 }
                 return new eventEmitter.EventEmitter();
             },
             statusCode: 200
         }
-        
+
         function parseDuration(duration: string): number {
             if (!duration) {
                 return 0;
             }
-            
-            var parts = duration.match("(\\d\\d):(\\d\\d):(\\d\\d).(\\d\\d\\d)");  
-            return parseInt(parts[1])*60*60*1000 + parseInt(parts[2])*60*1000 + parseInt(parts[3])*1000 + parseInt(parts[4]);    
+
+            var parts = duration.match("(\\d\\d):(\\d\\d):(\\d\\d).(\\d\\d\\d)");
+            return parseInt(parts[1]) * 60 * 60 * 1000 + parseInt(parts[2]) * 60 * 1000 + parseInt(parts[3]) * 1000 + parseInt(parts[4]);
```

Comment:
This could be re-parenthesized so that the * 60's and * 1000's aren't repeated for each value.
i.e. ((((parseInt(parts[1]) * 60) + parseInt(parts[2])) * 60) + parseInt(parts[3])) * 1000 + parseInt(parts[4])